src/gluonnlp/loss/label_smoothing.py

```diff
@@ -133,7 +100,7 @@ def hybrid_forward(self, F, inputs, units=None): # pylint: disable=arguments-dif
 
         Parameters
         ----------
-        F
+        F: mx.sym or mx.nd
```

Comment:
I believe we typically omit `F` in docstrings (e.g., see `activation_regularizer.py`), because the forward pass is invoked without passing `F` explicitly (it is done behind the scenes).